coalib/core/DependencyTracker.py

```diff
@@ -138,3 +138,25 @@ def check_circular_dependencies(self):
         traverse_graph(
             self._dependency_dict.keys(),
             lambda node: self._dependency_dict.get(node, frozenset()))
+
+    @property
+    def all_dependencies_resolved(self):
```

Comment:
From @SanketDG:

> I somehow feel that since this something that is mutated based on operation of other functions, it should be a function.
>
> properties are properties, features, attributes of an object.
>
> also, a name like are_dependencies_resolved() looks more suitable.

1. imo it describes a state, so properties should be suitable.
2. Hmm agree, `are_dependencies_resolved` sounds better :+1: